src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/DropIndexCommand.java

```diff
@@ -1,24 +1,34 @@
 package io.stargate.sgv2.jsonapi.api.model.command.impl;
 
 import com.fasterxml.jackson.annotation.JsonTypeName;
-import io.stargate.sgv2.jsonapi.api.model.command.CollectionCommand;
+import io.stargate.sgv2.jsonapi.api.model.command.KeyspaceCommand;
 import io.stargate.sgv2.jsonapi.api.model.command.NoOptionsCommand;
+import jakarta.annotation.Nullable;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-// TODO, hide table feature detail before it goes public,
-// https://github.com/stargate/data-api/pull/1360
-// @Schema(description = "Command that drops an index for a column.")
+@Schema(description = "Command that drops an index for a column.")
```

Comment:
Do we no longer need to hide the command?

Reply:
We have not added the command to anyOf schema in KeyspaceResource. So I believe, it wouldn't show up.

Reply:
reading this did make me realize we duplciate the command names here, added ticket to use the new enum https://github.com/stargate/data-api/issues/1592

Reply:
On this, and for consistency, we do have CreateIndex and others added to CollectionCommand such as `@JsonSubTypes.Type(value = CreateIndexCommand.class),`

Do we have a consistent approach for this ?